.github/scripts/code_style.sh

```diff
@@ -0,0 +1,54 @@
+#! /bin/bash
+
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+set -o errexit # Exit if command failed.
+set -o nounset # Exit if variable not set.
+set -o pipefail # Exit if pipe failed.
+
+# Pass anything as a first parameter to fix code style problems
+FIX=${1:-0}
+
+# Version 0.67 is the source of truth
+if ! [ -x "$(command -v uncrustify)" ]; then
+    git clone https://github.com/uncrustify/uncrustify.git
+    cd uncrustify
+    git checkout uncrustify-0.67
+    mkdir build
+    cd build
+    cmake ..
+    cmake --build .
+    sudo make install
+    cd ../..
+fi
+
+if [ $FIX -ne 0 ]; then
```

Comment:
generally making explicit check is good thing... by doing -ne 0 increase the number of possible value hold true.

usage() {
    echo "${0} [check|fix]" 1>&2
    exit 1
}


FIX=${1:-"check"}


if [[ "$FIX" == "check" ]]; then


elif [[ "$FIX" == "check" ]] then

else
 usage();
fi